components/title/basic/src/index.js

```diff
@@ -0,0 +1,51 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+
+/**
+ * Builds the modifier className to apply to the component following the SUIT naming convention (sui-TitleBasic--modifierInCamelCase).
+ * @param alignProp {string} property value for Title alignment.
+ * @returns {string} SUIT formatted classname.
+ */
+const _buildAlignClassName = (alignProp) => {
+  return alignProp ? 'sui-TitleBasic--align' + alignProp.charAt(0).toUpperCase() + alignProp.slice(1) : ''
```

Comment:
Since you have a list of valid align values, why not have a map lookup instead of slices?

I think will be a bit faster? not sure and I don't think really important though